Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -247,33 +196,37 @@ public Task InterceptAsync(HttpRequestMessage request, CancellationToken taskCan
             new GoogleCredential(computeCredential ?? new ComputeCredential());
 
         /// <summary>
-        /// <para>Returns <c>true</c> only if this credential type has no scopes by default and requires
-        /// a call to <see cref="CreateScoped(IEnumerable{string})"/> before use.</para>
-        ///
-        /// <para>Credentials need to have scopes in them before they can be used to access Google services.
-        /// Some Credential types have scopes built-in, and some don't. This property indicates whether
-        /// the Credential type has scopes built-in.</para>
-        /// 
+        /// <para>
+        /// Returns <c>true</c> only if this credential type supports explicit scopes
+        /// and no explicit scopes has been set. Explicit scopes may be set by calling
```

Comment:
has => have

I'd also say "A credential with scopes set may be created by calling [...]" otherwise it sounds like it's going to modify the existing credentials.